pandas/core/series.py

```diff
@@ -1885,10 +1885,109 @@ def update(self, other):
     # ----------------------------------------------------------------------
     # Reindexing, sorting
 
-    @Appender(generic._shared_docs['sort_values'] % _shared_doc_kwargs)
     def sort_values(self, axis=0, ascending=True, inplace=False,
                     kind='quicksort', na_position='last'):
+        """
+        Sort by the Series values.
+
+        Sort (or order) a Series into ascending or descending order by some criterion.
+
+        Parameters
+        ----------
+        axis : {0 or ‘index’}, default 0
+            Axis to direct sorting.
+        ascending : bool, default True
+            If  `True` sort values into ascending order, otherwise descending.
+        inplace : bool, default False
+            If True, perform operation in-place.
+        kind : {‘quicksort’, ‘mergesort’ or ‘heapsort’}, default ‘quicksort’
+            Choice of sorting algorithm. See also ndarray.np.sort [1]_ for more information. `mergesort` is the only stable
```

Comment:
you can use `` :func:`np.sort` `` as a syntax to create this reference automatically (then you don't need the `[1]_`